backend/api/tests.py

```
@@ -0,0 +1,43 @@
+class AuthTestCase(APITestCase):
+
+    def setUp(self):
+        self.user = Users.objects.create(
+            name="Test User",
+            email="test@example.com",
+            password=make_password("testpass123")
+        )
+        self.register_url = "/api/register/"
+        self.login_url = "/api/login/"
+
+    def test_register_user(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_register_user`

Add return type annotation: `None`

(ANN201)